recipes/r-pathfindr/meta.yaml

```diff
@@ -0,0 +1,128 @@
+{% set version = '2.1.0' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-pathfindr
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/pathfindR_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/pathfindR/pathfindR_{{ version }}.tar.gz
+  sha256: 340e58b36e16bc4b516064b57e0c0f22273961327dcd97701a79003fdc2cef46
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+
+  noarch: generic
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+    - openjdk 8.*
+
+  host:
+    - r-base
+    - openjdk 8.*
+    - bioconductor-annotationdbi
+    - r-dbi
+    - bioconductor-keggrest
+    - bioconductor-kegggraph
+    - r-r.utils
+    - r-doparallel
+    - r-foreach
+    - r-fpc
+    - r-ggplot2
+    - r-ggraph
+    - r-ggupset
+    - r-igraph
+    - r-knitr
+    - r-magick
+    - r-msigdbr
+    - bioconductor-org.hs.eg.db
+    - r-pathfindr.data >=2.0
```

Comment:
How come that this can't be part of the actual package?

Reply:
CRAN didn’t allow updates as the package size got bigger and suggested another package for only data that’d be less frequently updated. Thus, we have pathfindR.data as a dependency